privacy.tex

```diff
@@ -182,17 +182,16 @@ \subsection{Which third-party service providers process my personal information?
 
 "Third parties" includes third-party service providers (including contractors and designated agents) and other entities within our group. 
 \begin{itemize}
-\item We will share personal information with the \textbf{University of Southampton Students’ Union} as part of being an affiliated club or society, in order to process your membership, and to run the club or society.
+\item We will share personal information with the \textbf{University of Southampton Students’ Union} as part of being an affiliated club or society, in order to process your membership, manage our mailing list, and to run the club or society.
 \item We share some of your personal information with the \textbf{University of Southampton}, for access to physical and networked resources.
-\item We share some of your personal information with the \textbf{University of Southampton, School of Electronics and Computer Science}, through which we run our mailing list. Further details of your membership to this list can be found at \url{http://mailman.ecs.soton.ac.uk/mailman/listinfo/sucss}.
 \end{itemize}
 
 \subsection{How secure is my information with third-party service providers}
 
 All our third-party service providers and other entities in the group are required to take appropriate security measures to protect your personal information under the general law or in line with our policies. We do not allow our third-party service providers to use your personal data for their own purposes. We only permit them to process your personal data for specified purposes and in accordance with our instructions.
 
 \section{Data retention}
-We will only retain your personal information for as long as necessary to fulfil the purposes we collected it for, and usually for [number of years] after your membership ceases. This may include for the purposes of satisfying any legal, accounting, or reporting requirements. To determine the appropriate retention period for personal data, we consider the amount, nature, and sensitivity of the personal data, the potential risk of harm from unauthorised use or disclosure of your personal data, the purposes for which we process your personal data and whether we can achieve those purposes through other means, and the applicable legal requirements. 
+We will only retain your personal information for as long as necessary to fulfil the purposes we collected it for, and usually for 1 year after your membership ceases. This may include for the purposes of satisfying any legal, accounting, or reporting requirements. To determine the appropriate retention period for personal data, we consider the amount, nature, and sensitivity of the personal data, the potential risk of harm from unauthorised use or disclosure of your personal data, the purposes for which we process your personal data and whether we can achieve those purposes through other means, and the applicable legal requirements. 
```

Comment:
We dont do this, flag tracker data persists and so does IDM.
Probably should say we keep email, username data and discord. But this can eb requested to be delted